lib/liquid/tags/capture.rb

```diff
@@ -22,11 +22,11 @@ def initialize(tag_name, markup, options)
       end
     end
 
-    def render(context)
-      output = super
+    def render(context, output = '')
+      super
       context.scopes.last[@to] = output
-      context.resource_limits.assign_score += output.length
-      ''.freeze
+      context.resource_limits.assign_score = output.length
```

Comment:
Note to self for later: This looks like a logic bug.

Reply:
yea that doesn't look right